src/py21cmfast/drivers/lightcone.py

```diff
@@ -459,9 +459,18 @@ def _run_lightcone_from_perturbed_fields(
     ):
         # Save mean/global quantities
         for quantity in global_quantities:
-            lightcone.global_quantities[quantity][iz] = np.mean(
-                getattr(coeval, quantity)
-            )
+            if quantity == "log10_mturn_acg":
+                lightcone.global_quantities[quantity][iz] = (
+                    coeval.ionized_box.log10_Mturnover_ave
+                )
+            elif quantity == "log10_mturn_mcg":
+                lightcone.global_quantities[quantity][iz] = (
```

Comment:
Why can't we treat these like other globals and just take the mean of the output field?

Reply:
These fields aren't part of the output structs at the moment. Currently they are hidden in the backend, with the exception of the `XraySourcebox` which saves the MCG part when halos are used.

Reply:
Ahh I see. No probs.